app/assets/javascripts/discourse/app/routes/build-private-messages-route.js

```diff
@@ -65,6 +77,17 @@ export default (inboxType, path, filter) => {
         group: null,
         inbox: inboxType,
       });
+
+      let ascending = userTopicsListController.ascending;
+      if (ascending === "true") {
+        ascending = true;
+      } else if (ascending === "false") {
+        ascending = false;
+      }
+      userTopicsListController.setProperties({
```

Comment:
I think this could all be shortened to:

```javascript
userTopicsListController.setProperties({
  ascending: !!userTopicsListController.ascending
});

Reply:
No, I didn't want to update the value when it's other than `"true"` or `"false"` (for ex: `null`). If I try then it's not displaying the proper sort icon in certain cases.